.travis.yml

```diff
@@ -1,5 +1,22 @@
 language: ruby
 
+sudo: false
+
+cache:
+  directories:
+    - ../bundle
```

Comment:
You may want to do `cache: bundler` here rather than listing the directory.  Travis has special bundler support where it knows to run `bundle clean` before backing up the bundle so it doesn't grow indefinitely.

For the other repos we changed to specifying the directory specifically to avoid the `bundle clean` travis was doing, because I found that, for example, when running the rspec-mocks build, the "run the rspec-core and rspec-rails spec suites" parts would take a while because it would _always_ have to install fresh gems and they weren't in the cache.  This was happening because travis was running `bundle clean` (due to our `cache: bundler` config) and running `bundle clean` against rspec-mocks was pruning gems installed for the other repos like rspec-core and rspec-rails.

In your case, if you're not running the spec suites for the other repos (which I doubt -- given they don't depend on rspec-rails, why would you?) then `cache: bundler` is probably a better option.